acceptance/tests/reports/submission.rb

```diff
@@ -32,7 +32,7 @@ def query_last_report_time_on(agent)
       require "json"
 
       puppetdb_url = URI("http://localhost:8080/v3/reports")
-      puppetdb_url.query = URI.escape(%Q{query=["=","certname","#{agent}"]})
+      puppetdb_url.query = CGI.escape(%Q{query=["=","certname","#{agent}"]})
```

Comment:
I'm actually not sure if this is correct given `CGI.escape` also encodes `=` which is likely undesirable here (but depends on the puppetserver implementation):

```
[2] pry(main)> CGI.escape(%Q{query=["=","certname","foo.bar.com"]})
=> "query%3D%5B%22%3D%22%2C%22certname%22%2C%22foo.bar.com%22%5D"

[3] pry(main)> URI.escape(%Q{query=["=","certname","foo.bar.com"]})
=> "query=[%22=%22,%22certname%22,%22foo.bar.com%22]"
```

Though either way CGI.unescape produces the correct original value:

```
[8] pry(main)> query = %Q{query=["=","certname","foo.bar.com"]}
=> "query=[\"=\",\"certname\",\"foo.bar.com\"]"
[9] pry(main)> URI.unescape(URI.escape(query)) == query
=> true
[10] pry(main)> URI.unescape(CGI.escape(query)) == query
=> true
```